src/main/telemetry/crsf.c

```diff
@@ -388,6 +391,12 @@ void crsfSendMspResponse(uint8_t *payload)
 static void processCrsf(void)
 {
     static uint8_t crsfScheduleIndex = 0;
+#ifdef USE_GPS
+    if (!gps_telemetry_added && (featureIsEnabled(FEATURE_GPS) || sensors(SENSOR_GPS))) {
```

Comment:
`featureIsEnabled(FEATURE_GPS) || sensors(SENSOR_GPS)` seems to be redundant. `sensors(SENSOR_GPS)` can only be true if ``featureIsEnabled(FEATURE_GPS)` is true.

Reply:
OK, want me to use ```sensors(SENSOR_GPS)``` or ```featureIsEnabled(FEATURE_GPS)```?